### PR TITLE
Rename/wordpress classicpress adminfolder

### PR DESCRIPTION
**Pull request by senlin**
* file `src/wp-admin/includes/misc.php` contains rewrite rules with which I have done nothing as I was unsure: L678, L708, L749
* everywhere where WP version numbers are mentioned, I have not done renaming: `src/wp-admin/js/customize-controls.js` L5205, `src/wp-admin/includes/upgrade.php` 28 lines with format `Execute changes made in WordPress x.y.`, `src/wp-admin/includes/schema.php` 17 lines to do with Roles that were created/modified during certain WP versions

EDIT: TravisCI fails, will have to get back at this later.

### CODE REVIEW

**Comment by nylen** [comment]
## Rewrite rules

> file `src/wp-admin/includes/misc.php` contains rewrite rules with which I have done nothing as I was unsure

https://github.com/ClassicPress/ClassicPress/blob/94703a6809d583762d8e5b5e9c8a96126cc39594/src/wp-admin/includes/misc.php#L678

https://github.com/ClassicPress/ClassicPress/blob/94703a6809d583762d8e5b5e9c8a96126cc39594/src/wp-admin/includes/misc.php#L708

https://github.com/ClassicPress/ClassicPress/blob/94703a6809d583762d8e5b5e9c8a96126cc39594/src/wp-admin/includes/misc.php#L749

This is an interesting one... in order to properly test it, we need an IIS7 server set up.  I'll open a follow-up issue for this.

## WP history

>everywhere where WP version numbers are mentioned, I have not done renaming: src/wp-admin/js/customize-controls.js L5205, src/wp-admin/includes/upgrade.php 28 lines with format Execute changes made in WordPress x.y., src/wp-admin/includes/schema.php 17 lines to do with Roles that were created/modified during certain WP versions

https://github.com/ClassicPress/ClassicPress/blob/94703a6809d583762d8e5b5e9c8a96126cc39594/src/wp-admin/js/customize-controls.js#L5205

https://github.com/ClassicPress/ClassicPress/blob/94703a6809d583762d8e5b5e9c8a96126cc39594/src/wp-admin/includes/upgrade.php#L626

https://github.com/ClassicPress/ClassicPress/blob/94703a6809d583762d8e5b5e9c8a96126cc39594/src/wp-admin/includes/schema.php#L614

I agree with this, since the history of WP is the history of ClassicPress also :+1: 

## Next steps

Let's get the Travis tests passing and merge this as an initial step.

I noticed something else we'll need to do - change all the `@since` tags to something like `@since WP-X.X.X`:

https://github.com/ClassicPress/ClassicPress/blob/94703a6809d583762d8e5b5e9c8a96126cc39594/src/wp-admin/includes/schema.php#L616

I'll open a separate issue for this too.

**Comment by nylen** [comment]
>Were you lured away from your desk before you saved your new draft design? Fear not, when you return, ClassicPress 4.9 will politely ask whether or not you&#8217;d like to save your unsaved changes.

Before merging this, I'm going to selectively undo a few of these changes that reference specific, past WordPress versions.

File: src/wp-admin/about.php [hunk 1]
```diff
@@ -2,26 +2,26 @@
...
 list( $display_version ) = explode( '-', get_bloginfo( 'version' ) );
 
 include( ABSPATH . 'wp-admin/admin-header.php' );
 ?>
 	<div class="wrap about-wrap full-width-layout">
-		<h1><?php printf( __( 'Welcome to WordPress&nbsp;%s' ), $display_version ); ?></h1>
+		<h1><?php printf( __( 'Welcome to ClassicPress&nbsp;%s' ), $display_version ); ?></h1>
 
-		<p class="about-text"><?php printf( __( 'Thank you for updating to the latest version! WordPress %s will smooth your design workflow and keep you safe from coding errors.' ), $display_version ); ?></p>
+		<p class="about-text"><?php printf( __( 'Thank you for updating to the latest version! ClassicPress %s will smooth your design workflow and keep you safe from coding errors.' ), $display_version ); ?></p>
```
[the remark this answers is not in the document]
Comment (nylen) [reply src/wp-admin/about.php]: There are also a number of strings like this:  `/* translators: 1: ClassicPress version number, 2: plural number of bugs. */`

Let's leave these as ClassicPress for now, but I think later on we'll want to make some pretty significant modifications to the About page, and make sure the content and layout of this page makes sense with the history of the fork.

File: src/wp-admin/about.php [hunk 2]
```diff
@@ -334,9 +334,9 @@
 				?>
 			</h2>
 			<p><?php printf(
-				__( 'WordPress is working on a new way to create and control your content and we&#8217;d love to have your help. Interested in being an <a href="%s">early tester</a> or getting involved with the Gutenberg project? <a href="%s">Contribute on GitHub</a>.' ),
+				__( 'ClassicPress is working on a new way to create and control your content and we&#8217;d love to have your help. Interested in being an <a href="%s">early tester</a> or getting involved with the Gutenberg project? <a href="%s">Contribute on GitHub</a>.' ),
```
Comment (nylen) [inline src/wp-admin/about.php]: :thinking:

File: src/wp-admin/credits.php [hunk 3]
```diff
@@ -117,7 +117,7 @@
 
 // These are strings returned by the API that we want to be translatable
 __( 'Project Leaders' );
-__( 'Core Contributors to WordPress %s' );
+__( 'Core Contributors to ClassicPress %s' );
```
Comment (nylen) [inline src/wp-admin/credits.php]: Later, we'll also want to re-do this page to account for the fork.

File: src/wp-admin/includes/ajax-actions.php [hunk 4]
```diff
@@ -3338,7 +3338,7 @@ function wp_ajax_generate_password() {
 }
 
 /**
- * Ajax handler for saving the user's WordPress.org username.
+ * Ajax handler for saving the user's ClassicPress.net username.
```
Comment (nylen) [inline src/wp-admin/includes/ajax-actions.php]: Fine for now, but flagging this because it isn't a thing yet.

**Comment by nylen** [comment]
Nice job, @senlin, and thanks for getting this started.  It's good for an initial step.

File: src/wp-admin/about.php (hunk 2 above)
Comment (senlin) [reply src/wp-admin/about.php]: I was surprised that that was still there, and I see you have addressed it with #28

**Pull request merged**